src/app/FailSafeContext.h

```diff
@@ -82,7 +82,7 @@ class FailSafeContext
 
     bool MatchesFabricIndex(FabricIndex accessingFabricIndex) const
     {
-        VerifyOrDie(mFailSafeArmed);
+        VerifyOrDie(IsFailSafeArmed());
```

Comment:
Doesn't this cause a recursive problem where is IsFailsafeArmed calls this function? Or are there two versions of that function?

Reply:
There are two IsFailSafeArmed(): one takes no args and checks whether the failsafe is armed at all, and one that takes a fabric index and checks whether the failsafe is armed for that specific fabric index.

This is called from the version that takes a fabric index, and it's calling the one that does not take a fabric index.